Apps/WebClient/src/ClientApp/src/services/restConfigService.ts

```diff
@@ -5,9 +5,10 @@ import { IConfigService, IHttpDelegate, ILogger } from "@/services/interfaces";
 import ErrorTranslator from "@/utility/errorTranslator";
 
 export class RestConfigService implements IConfigService {
-    private readonly CONFIG_BASE_URI: string = "/configuration";
     private logger;
+    private readonly CONFIG_BASE_URI: string = "configuration" ;
     private http;
+    private baseUri = import.meta.env.VITE_WEB_CLIENT_BASE_URI || "/" ;
```

Comment:
```suggestion
    private readonly CONFIG_BASE_URI: string = "configuration";
    private http;
    private baseUri = import.meta.env.VITE_WEB_CLIENT_BASE_URI || "/";
```

Just removing the extra spaces.  (This should happen automatically on save when using VS Code and opening the workspace.)